cargo-n64/Cargo.toml

```
@@ -12,8 +12,8 @@ edition = "2018"
 
 [dependencies]
 byteorder = "1.3"
-cargo-xbuild = "0.5"
-colored = "1.9"
+cargo-xbuild = "0.6"
+colored = "2"
```

For consistency, please use `2.0`

```suggestion
colored = "2.0"
```